src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -204,7 +208,7 @@ public String getResourceURI(IResource resource) {
 		if(ResourceKind.LIST.equals(resource.getKind())) throw new UnsupportedOperationException("Generic create operation not supported for resource type 'List'");
 		try {
 			namespace = ResourceKind.PROJECT.equals(resource.getKind()) ? "" : namespace;
```

Comment:
I think this is code left from the early impl of the class when the server api was flaky. Using null might make more sense here but I have not thought about it in depth